server/cluster/unsafe_recovery_controller.go

```diff
@@ -112,6 +116,7 @@ func (u *unsafeRecoveryController) RemoveFailedStores(failedStores map[uint64]st
 		}
 		u.storeReports[s.GetID()] = nil
 	}
+	u.cluster.PauseOrResumeScheduler("all", timeout)
```

Comment:
You mean those who have been triggered already? No

Reply:
`PauseOrResumeScheduler` only prevents the schedulers from generating the scheduling operator. The other things like checkers also will generate the operator.

Reply:
Please take a look at the new code added in `server/schedule/checker/checker_controller.go`

Reply:
BTW, do you feel comfortable we do the same to the schedulers? i.e. checking if `unsafeRecoveryController.IsRunning()` instead of pause schedulers for a fixed amount of time? We may add another cmd in a separate PR to interrupt unsafe recovery process, if you also worry about unsafe recovery hangs forever.

Reply:
For now, it's ok to me. But I think it's a common feature to pause all scheduling. It's better not to rely on the unsafe recovery.

Reply:
PTAL